lib/rock_rms/version.rb

```diff
@@ -1,3 +1,3 @@
 module RockRMS
-  VERSION = '9.9.0'.freeze
+  VERSION = '9.10.0'.freeze
```

Comment:
I'll bump the version when I release a new gem. No need to do it as part of a PR.